### PR TITLE
Reference to latest version of Microsoft.IdentityModel.Clients.ActiveDirectory

### PR DESCRIPTION
**Pull request by bikamani**
Reference to latest version of Microsoft.IdentityModel.Clients.ActiveDirectory as per security alert

### CODE REVIEW

File: e2e/test/E2ETests.csproj [hunk 1]
```diff
@@ -139,4 +139,4 @@
     </PackageReference>
   </ItemGroup>
 
-</Project>
\ No newline at end of file
+gi</Project>
```
Comment (abhipsaMisra) [inline e2e/test/E2ETests.csproj]: "gi"?

Comment (bikamani) [reply e2e/test/E2ETests.csproj]: Not sure how did that come here. Thanks for noticing. updated.

**Comment by bikamani** [comment]
> Thanks @bikamani ! This is required in `master` too I believe.

There is a PR out there for master as well.

**Pull request merged**